cryptopan/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+<h1>Changelog</h1>
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning]https://semver.org/spec/v2.0.0.html
+
+<h1>0.1.0-PRE (2022-01-23)</h1>
+<h2>Added</h2>
```

Comment:
You should use `#` and `##` instead of the HTML tags, see the other crate for reference.